.github/workflows/Tests.yml

```diff
@@ -0,0 +1,72 @@
+name: Tests
+
+on: [push]
+
+jobs:
+  UnitTests-Debug-x64:
+    name: Unit tests - Debug-x64
+    runs-on: windows-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup MSBuild.exe
+        uses: microsoft/setup-msbuild@v1.0.2
+
+      - name: Setup NuGet
+        uses: nuget/setup-nuget@v1
+
+      - name: Restore all the packages
+        run: nuget restore SharpVector.sln
+
+      - name: Build Debug x64
+        run: msbuild -p:Configuration=Debug -p:Platform=x64 SharpVector.sln -target:Tests
+
+      - name: Setup VSTest Path
+        uses: darenm/Setup-VSTest@v1
+
+      - name: Run unit tests
+        run: vstest.console.exe Build\Tests\Debug-x64\Tests.dll
+
+
+  Examples-Debug-x64:
+    name: Examples Debug-x64

```

Comment:
The same step is already performed in the Build-MSBuild workflow.
To avoid breaking DRY *Examples* executable should just be cached and launched during Examples tests.